stdlib/universe/universe.flux

```diff
@@ -4643,3 +4643,187 @@ toTime = (tables=<-) => tables |> map(fn: (r) => ({r with _value: time(v: r._val
 // tags: date/time
 //
 today = () => date.truncate(t: now(), unit: 1d)
+
+// yesterday finds the starting and stopping point of yesterday truncated to the day.
+//
+// ## Examples
+//
+// ### Returns a recording containing the timestamp of the start and the stopping point of yesterday
+// ```no_run
+// option now = () => 2022-01-02T13:45:28Z
+//
+// yesterday()
+// // Returns {start: 2022-01-01T00:00:00.000000000Z, stop: 2022-01-02T00:00:00.000000000Z}
+// ```
+//
+// ### Query data from yesterday
+// ```no_run
+// ret = yesterday()
+// from(bucket: "example-bucket")
+//     |> range(start: ret.start, stop: ret.stop )
+// ```
+//
+// ## Metadata
+// introduced: NEXT
+// tags: date/time
+
```

Comment:
nit, this small space means that the comment above doesn't get associated with the yesterday function.

Reply:
Cleaned that one up now